src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -50,7 +50,16 @@ internal ActorRuntime(ActorRuntimeOptions options, ILoggerFactory loggerFactory,
             // Revisit this if actor initialization becomes a significant source of delay for large projects.
             foreach (var actor in options.Actors)
             {
-                var daprInteractor = new DaprHttpInteractor(clientHandler: null, httpEndpoint: options.HttpEndpoint, apiToken: options.DaprApiToken, requestTimeout: null);
+                IDaprInteractor daprInteractor;
+                if (options.useGrpc) {
+                daprInteractor = new DaprInteractorBuilder()
+                    .UseGrpcEndpoint(options.GrpcEndpoint)
+                    .UseDaprApiToken(options.DaprApiToken)
+                    .UseGrpcChannelOptions(options.GrpcChannelOptions)
+                    .Build();                           
```

Comment:
Needs an indent but I also feel it shouldn't use a builder.

If you do want to use a builder, I think it should take in the `options.UseGrpc` field and return the appropriate interactor. The benefit of that being an interface is that we don't need to know the actual implementation.